php/main.php

```diff
@@ -9,76 +8,79 @@
 $password = "root";
 $database = "homeless";
 
+$conn = mysqli_connect($servername, $username, $password, $database);
+
+if ($_SERVER['REQUEST_METHOD'] === 'POST'){
+
+    if (isset($_POST['finger_status'])){
 
-if ($_SERVER['REQUEST_METHOD'] === 'POST') {
-  if (isset($_POST['finger_status'])){
-    $status = $_POST['finger_status'];
-    if ($status === "success"){
-      $conn = mysqli_connect($servername, $username, $password, $database);
-      $sql = "UPDATE coreData SET state='3' WHERE State='2'";
-      $conn->query($sql);
+        $status = $_POST['finger_status'];
+
+        if ($status === "success"){
+            $sql = "UPDATE coreData SET state='3' WHERE State='2'";
+        }
+        else if ($status == "failure"){
+            $sql = "UPDATE coreData SET Error=Error + 1 WHERE State='2'";
+        }
```

Comment:
We could make this `else` rather than `else if` to handle any other cases

Reply:
Sure